packages/actor-http-memento/lib/ActorHttpMemento.ts

```diff
@@ -0,0 +1,64 @@
+import { ActorHttp, IActionHttp, IActorHttpOutput } from "@comunica/bus-http";
+import { ActionContext, IActorArgs, IActorTest, Mediator } from "@comunica/core";
+import "isomorphic-fetch";
+import * as parseLink from 'parse-link-header';
+
+/**
+ * A comunica Memento Http Actor.
+ */
+export class ActorHttpMemento extends ActorHttp {
+
+  public readonly mediatorHttp: Mediator<ActorHttp,
+    IActionHttp, IActorTest, IActorHttpOutput>;
+
+  constructor(args: IActorHttpMementoArgs) {
+    super(args);
+  }
+
+  public async test(action: IActionHttp): Promise<IActorTest> {
+    if (!(action.context && action.context.has('datetime') && action.context.get('datetime') instanceof Date)) {
+      throw new Error('This actor only handles request with a set valid datetime.');
+    }
+    if (action.init && new Headers(action.init.headers || {}).has('accept-datetime')) {
+      throw new Error('The request already has a set datetime.');
+    }
+    return true;
+  }
+
+  public async run(action: IActionHttp): Promise<IActorHttpOutput> {
+    const datetime: Date = action.context && action.context.get('datetime');
+
+    // 1. Create ActionHttp
```

Comment:
I would remove these 1., 2., 3. 4., 5. comments, they don't really add any information next to the code IMO.